docs/whatsnew/4.1.rst

```diff
@@ -274,6 +274,21 @@ Support for ASDF serialization of models
 All models (excluding model sets) can be written to an ASDF file. Constraints
 of type ``fixed`` and ``bounds`` can also be serialized.
 
+
+.. _whatsnew-4.1-light-defelction-fix:
+
+Realistic treatment of solar light deflection for nearby objects
+================================================================
+
+The machinery underlying the `astropy.coordinates` transformations was updated
+to correct an issue where coordinates were always assumed to be at infinite
+distance when computing the effect of light deflection by the sun.  While this
+change is undeniably more accurate, this will change previous results that may
```

Comment:
The "While this change..." sentence reads a bit weirdly. How about: "With this change, light deflection by the Sun is accurate also for solar system objects. Even for those, the results will change only slight be slightly (by 3 arcsec in the absolute worst case).
```
I don't think we have to warn about "may have already corrected" - anybody who knowingly did do a post-astropy correction will understand they need to change their code (and, really, they should have pinged us upstream in the first place!)